pkg/operation/seed/seed.go

```diff
@@ -393,6 +393,11 @@ func BootstrapCluster(ctx context.Context, k8sGardenClient, k8sSeedClient kubern
 			return err
 		}
 
+		lokiValues["priorityClass"] = map[string]interface{}{
+			"value": 100,
```

Comment:
This should be higher than the priority of `gardener-shoot-controlplane` (currently set to `100`), right?

Reply:
The mess up here is coming from the fact that there are two different pools: The etcd and the general one.
Both pools share the same priority classes. Here for example: gardener-shoot-controlplane is used from kube-apiserver and etcd pod. but in the same time both pods do not share the same nodes.

The Idea of Vladimir is to make the priorityclass of the Loki to be equal to the one of the ETCD (e.g. 100). In this way Loki pods will not be preempted by the etcd pods

Reply:
There is no need loki to preempt or be places ahead of the etcd in the scheduling queue. Onece loki find its place on a node it will not be preempted by other etcd pod. So 100 is enough.